ml-pipeline/Dockerfile

```diff
@@ -1,23 +1,17 @@
 # This is a dockerfile for an image with kubectl and ksonnet installed
 # Available at gcr.io/ml-pipeline/bootstrapper
 
-FROM ubuntu:18.04
+FROM google/cloud-sdk
 
 RUN apt-get update -y
 RUN apt-get install -y curl
 
-# Install kubectl
-RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl
-RUN chmod +x ./kubectl
-RUN mv kubectl /usr/local/bin/
-
 # Install ksonnet
 RUN curl -LO https://github.com/ksonnet/ksonnet/releases/download/v0.11.0/ks_0.11.0_linux_amd64.tar.gz
```

Comment:
it will be nicer if the version of ksonnet is defined as an ARG variable, it will make it easier to update and test diff versions of it
i.e 
ARG KS_VERSION="0.11.0"
RUN curl -LO https://github.com/ksonnet/ksonnet/releases/download/v$KS_VERSION/ks_$KS_VERSION_linux_amd64.tar.gz

Reply:
thanks. done :)